app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :likes, foreign_key: 'post_id'
   has_many :comments, foreign_key: 'post_id'
 
+  validates :title, presence: true, length: { maximum: 255 }
```

Comment:
- [ ] Hey there! 🌟 Great job on adding validation for the post title! 🎉 However, there's a tiny tweak needed to align with the project requirements. The maximum character limit for the title should be 250, but in the current implementation, it's set to 255. 🚨 Let's adjust that to ensure we're spot-on with the project specifications! 🛠️ Thanks a bunch! 👍

![Screenshot from 2023-12-01 20-03-36](https://github.com/Chimoneg27/Rails-Blog-App/assets/112261237/58b00de8-3cdd-4a7b-8909-bc38440ebc95)